src/engine/surface_collision.c

```diff
@@ -646,11 +647,14 @@ f32 find_room_floor(f32 x, f32 y, f32 z, struct Surface **pfloor) {
 s32 get_room_at_pos(f32 x, f32 y, f32 z) {
     if (gCurrentArea->surfaceRooms != NULL) {
         struct Surface *floor;
+
         find_room_floor(x, y, z, &floor);
-        if (floor) {
+
+        if (floor != NULL) {
```

Comment:
cmon these formatting changes arent necessary lol i like truthiness better

Reply:
Nowhere else in the repo has `if (floor) {`